app/src/main/java/ch/epfl/sdp/peakar/general/SettingsMapActivity.java

```diff
@@ -48,6 +50,7 @@
 
     private Button downloadButton;
     private View loadingView;
+    public static OSMMap osmMap;
```

Comment:
Why has this to be static?

Reply:
This was code that was copy-pasted. This is now resolved. It was made static to be used in the instrumented tests but I changed the public static to a private identifier.